package.json

```diff
@@ -1,138 +1,119 @@
 {
   "name": "videojs-contrib-ads",
   "description": "A framework that provides common functionality needed by video advertisement libraries working with video.js.",
-  "main": "es5/plugin.js",
+  "main": "dist/videojs-contrib-ads.cjs.js",
+  "module": "dist/videojs-contrib-ads.es.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/videojs/videojs-contrib-ads.git"
   },
   "version": "5.0.1",
-  "author": "Brightcove",
+  "author": "Brightcove, Inc.",
   "engines": {
-    "node": ">= 0.8.0"
+    "node": ">=4.8.0"
   },
   "scripts": {
     "prebuild": "npm run clean",
     "build": "npm-run-all -p build:*",
-    "build:css": "npm-run-all build:css:sass build:css:bannerize",
-    "build:css:bannerize": "bannerize dist/videojs.ads.css --banner=scripts/banner.ejs",
-    "build:css:sass": "node-sass src/plugin.scss dist/videojs.ads.css --output-style=compressed --linefeed=lf",
-    "build:js": "npm-run-all build:js:babel build:js:browserify build:js:bannerize build:js:collapse build:js:uglify",
-    "build:js:babel": "babel src -d es5",
-    "build:js:bannerize": "bannerize dist/videojs.ads.js --banner=scripts/banner.ejs",
-    "build:js:browserify": "browserify . -g browserify-shim -s videojs-contrib-ads -o dist/videojs.ads.js",
-    "build:js:collapse": "bundle-collapser dist/videojs.ads.js -o dist/videojs.ads.min.js",
-    "build:js:uglify": "uglifyjs dist/videojs.ads.min.js --comments --mangle --compress --support-ie8 -o dist/videojs.ads.min.js",
-    "build:test": "babel-node scripts/build-test.js",
-    "clean": "rimraf dist test/dist es5 && mkdirp dist test/dist es5",
-    "lint": "vjsstandard src",
-    "start": "babel-node scripts/server.js",
+    "build:css": "npm-run-all build:css:sass build:css:bannerize build:css:copy",
+    "build:css:bannerize": "bannerize dist/videojs-contrib-ads.css --banner=scripts/banner.ejs",
+    "build:css:sass": "node-sass src/plugin.scss dist/videojs-contrib-ads.css --output-style=compressed --linefeed=lf",
+    "build:css:copy": "cpy dist/videojs-contrib-ads.css dist/ --rename=video.ads.css",
```

Comment:
There are three scripts like this that aim to preserve the old filenames (`video.ads.*`) vs. the more conventional names (`videojs-contrib-ads.*`) - `dist/` will contain _both_.

It was easier to do this instead of trying to make every piece of tooling adapt to the old filenames.